scripts/instance/mask_rcnn/requirements.sh

```diff
@@ -0,0 +1,10 @@
+pip install pybind11==2.5.0
+
+# Install NVIDIA COCO
+pip uninstall -y pycocotools
+pip --no-cache-dir --no-cache install \
+    'git+https://github.com/NVIDIA/cocoapi#egg=pycocotools&subdirectory=PythonAPI'
+
+# Install nightly GluonCV for now since changes in Mask RCNN is not compatible with stable pip release
+pip uninstall -y gluoncv
+pip install gluoncv==0.8.0b20200801
```

Comment:
is gluon cv gonna fix it in later release, and do we need to build with cython also?

Reply:
Once we have a 0.8.0 stable release or atleast a tag, we can use that. For now, it doesn't exist

Reply:
Added a stable version in a comment